src/databricks/labs/ucx/assessment/queries/all_tables.sql

```diff
@@ -0,0 +1,24 @@
+-- viz type=table, name=Table Types, columns=database,name,type,format,table_view,storage,is_delta,location
+-- widget title=Table Types, col=0, row=0, size_x=6, size_y=6
+SELECT `database`,
+       name,
+       object_type AS type,
+       UPPER(table_format) AS format,
+       CASE
+           WHEN object_type IN ("MANAGED",
+                                "EXTERNAL") THEN "TABLE"
+           ELSE "VIEW"
+       END AS table_view,
+       CASE
+           WHEN STARTSWITH(location, "/dbfs/")
+                AND NOT STARTSWITH(location, "/dbfs/mnt") THEN "DBFS ROOT"
+           WHEN STARTSWITH(location, "/dbfs/")
```

Comment:
First startswith is redundant

Reply:
See comment below.

Reply:
`STARTSWITH(location, "/dbfs/") AND STARTSWITH(location, "/dbfs/mnt")` is redundant by boolean logic.